### DISCUSSION TITLE
Next Release (F-Droid)

### DISCUSSION THREAD

**Discussion by Nickwasused** [twireapp/Twire#257] [General]
I am just going to forward this request from Twitter as the Author did not respond:
![firefox_SjKWa7gmM6](https://user-images.githubusercontent.com/34883496/137782480-bd9455aa-b1db-46bf-affe-6db006dd64d4.png)

He wants a new release to F-Droid since it's pretty outdated there. The last release was on January 23, 2021.
+ Related Issue #255

**Comment by samfundev** [comment]
I agree, we should get out a new release ASAP. I'd like to get #251 merged in before the release. Also @Nickwasused, I wasn't sure if I should merge your PRs after I reviewed them in case you wanted to add anything else to the PR. Feel free to merge them or let me know if I should merge those in.

**Reply by Nickwasused** [reply]
I wasn´t sure either if I can merge them after review so I just let the PR sit there for a while. But I can merge my PRs after you review them.

For #251 I am not sure if the issue is fixed I am still waiting for testing, but I am going to test it right now with the fix and without it.

**Reply by Perflyst** [reply]
> merge

I would say if a PR is approved, the author can merge it. If the PR is from a non member of course a member must merge.

**Reply by peepo5** [reply]
I agree, no point in changes if no one recieves updates
